4/john-wiegley-captions.srt

```diff
@@ -4225,16 +4230,16 @@ from something to string.
 
 986
 01:22:18,940 --> 01:22:20,000
-JW: Your right.
+JW: You're right.
 
 987
 01:22:31,760 --> 01:22:33,520
-OC: I think your alright using show.
+OC: I think we're OK using show.
 
 988
 01:22:34,941 --> 01:22:37,529
 Oh yes ! because we're
-defining a show of instance.
+defining a show instance.

```

Comment:
Maybe `show` should be `Show`, since they're talking about [Text.Show](https://hackage.haskell.org/package/base-4.8.0.0/docs/Text-Show.html) (I think).